src/System.Runtime.WindowsRuntime/src/System/Windows/Rect.cs

```diff
@@ -51,9 +51,9 @@ public struct Rect : IFormattable
                     double height)
         {
             if (width < 0)
-                throw new ArgumentException(nameof(width));
+                throw new ArgumentException(SR.Argument_NeedNonNegNum, nameof(width));
```

Comment:
@mellinoe, can you help validate these WindowsRuntime changes and suggest what @Clockwork-Muse should do here? (Or point to someone who can?)